### DISCUSSION TITLE
Is XLSB (Excel binary file format) format supported?

### DISCUSSION THREAD

**Discussion by slo-loris** [qax-os/excelize#1376] [Q&A]
Hi,

I went through the documentation of excelize, but wasnt able to figure out if it supports XLSB format. From the basic readme I understand it doesn't. If it does support I need to know how to write one.

PS: MS Excel opens XLSB file much faster and they are almost ~50% smaller in size compared to standard XLSX format.

Thanks in advance

**Comment by xuri** [comment]
Sorry, this library doesn't support this doc format currently.

**Reply by slo-loris** [reply]
Thanks for the clarification..

Are there any future plans to add the format?

**Reply by xuri** [reply]
Sorry, there is no plan to add this format support for this library recently.
